removeDuplicate.js

```diff
@@ -0,0 +1 @@
+const removeDuplicates =(array)=> [...new Set(array)];
```

Comment:
**Optional:** Can you consider removing this brackets on the array as this arrow function has only one parameter?